### ISSUE TITLE
PayPal Adaptive Payments Error Logging Fails

### ISSUE THREAD

**Issue by GoTeamScotch** [wcvendors/wcvendors#198]
This bug is about the PayPal Adaptive Payments SDK. Not sure if this should be posted at their github or here...

After installing WC Vendors on my site and after having a few customers check out (successfully), I noticed some new errors piling up in my php error log. The error is this:
```
PHP Warning:  error_log(PayPal.log): failed to open stream: Permission denied in [my wp-content path]\plugins\wc-vendors\classes\gateways\PayPal_AdvPayments\PayPal_AP\classes\adaptivepayments-sdk\lib\PPLoggingManager.php on line 58
```
And line 58 of that file is:
```
error_log( $this->loggerName . ": $message\n", 3, $this->loggerFile );
```
$this->loggerFile is defined in PPConfigManager.php on line 59 as simply "PayPal.log".
So it's trying to log an error in its own error log file. I don't see a path defined so it must be in the same directory as where it's being called (...adaptivepayments-sdk\lib\). The permissions on my server seem ordinary so I'm surprised that the file can't be written.

Long story short, can it just use error_log() as usual without specifying its own file? Basically just changing the above line 58 chunk to this:
```
error_log( $this->loggerName . ": $message\n" );
```
Edit: Covered in pull request: https://github.com/wcvendors/wcvendors/pull/201

**Comment by GoTeamScotch** [comment]
Issue resolved: https://github.com/wcvendors/wcvendors/pull/201

**Comment by bentasm1** [comment]
Thanks for your contributions!  It's nice to see you pushing new code and truly making WC Vendors a proper open source project. :)

**Comment by GoTeamScotch** [comment]
I'm happy to help! WC Vendors is a great plugin that saved me from having to migrate my store to a different shopping cart software (Magento) after the last vendor plugin I bought ($90), [IgniteWoo's Vendor Stores] (http://ignitewoo.com/woocommerce-extensions-plugins-themes/woocommerce-vendor-stores/), didn't work out. It's the least I can do. :+1:
